### ISSUE TITLE
All files scanned by amavis are marked as ***UNCHECKED***

### ISSUE THREAD

**Issue by thegrillmaster** [scVENUS/PeekabooAV#109]
All files scanned by amavis are marked as **** ***UNCHECKED*** even though Cuckoo has successfully completed the analysis.

## Steps to Reproduce

1.
2.
3.
4.

## Possible Implementation

Hello everyone, do any of you have any idea where the mistake is? I've been trying to find the bug for several hours, but can't find it. Because the analysis at Cuckoo runs without errors. It's very unfathomable to me. Enclosed the peekaboo.log amavis.log, cuckoo.log, analyzer.log.
Thank you very much for your help!

**Comment by thegrillmaster** [comment]
[peekaboolog.txt](https://github.com/scVENUS/PeekabooAV/files/3751013/peekaboolog.txt)
[cuckoolog.txt](https://github.com/scVENUS/PeekabooAV/files/3751014/cuckoolog.txt)
[analyzerlog.txt](https://github.com/scVENUS/PeekabooAV/files/3751015/analyzerlog.txt)
[amavislog.txt](https://github.com/scVENUS/PeekabooAV/files/3751016/amavislog.txt)

**Comment by Jack28** [comment]
TLDR: `end of analysis reached!`

This is because if the analysis fails we can not make any assumption about the file.

I'll point you to the right places. This behaviour can be configured here:
https://github.com/scVENUS/PeekabooAV/blob/master/ruleset.conf.sample#L156
You can find the corresponding logic here:
https://github.com/scVENUS/PeekabooAV/blob/master/peekaboo/ruleset/rules.py#L454
Peekaboo report doesn't match any of the amavis regexes so the mail is handed back and treated as `UNCHECKED`:
https://github.com/scVENUS/PeekabooAV/blob/master/docs/source/config.rst#amavisd

**Comment by Jack28** [comment]
Can I close this issue?

**Comment by thegrillmaster** [comment]
Many thanks for the answer and sorry for my late feedback. That made it work.
